code/game/machinery/launch_pad.dm

```diff
@@ -153,7 +146,10 @@
 
 	var/turf/source = target
 	var/list/log_msg = list()
-	log_msg += ": [key_name(user)] has teleported "
+	if(!isnull(inserter))
+		log_msg += ": [inserter] triggered a teleport"
```

Comment:
```suggestion
		log_msg += ": [key_name(inserter)] triggered a teleport"
```

Reply:
This wouldn't work like you expect, I think. Inserter is a string that contains a lot of information about the board (like who built it, what shell it uses, what id its linked, etc). Thought It'd be nicer to log all that, rather than only the name of who built the board.

Reply:
What `inserter` receives should be documented as being a string then.

Reply:
I'm confused, what do you mean by documented?

Reply:
get_creator() is a string of information, yes.